controllers/admin/AdminProductsController.php

```diff
@@ -4318,7 +4318,9 @@ public function initFormAttributes($product)
                     $attributes = Attribute::getAttributes($this->context->language->id, true);
                     foreach ($attributes as $k => $attribute) {
                         $attribute_js[$attribute['id_attribute_group']][$attribute['id_attribute']] = $attribute['name'];
-                        natsort($attribute_js[$attribute['id_attribute_group']]);
+                    }
+                    foreach (array_keys($attribute_js) as $id_attribute_group) {
+                        natsort($attribute_js[$id_attribute_group]);
```

Comment:
What do you think about

```
                   foreach ($attribute_js as &$attribute_group) {
                        natsort($attribute_group);
                   }
```

I haven't tested this code, but it should be easier to read as we directly modify the value, without going though the key.

Reply:
Yep, you're right, updated my PR